### PR TITLE
Revert "add def comp info for pytket-qiskit and update headings, info…

### PR DESCRIPTION
**Pull request by CalMacCQ**
Reverts CQCL/pytket-qiskit#3

Revert README.md back to original state

### CODE REVIEW

File: pytket/extensions/qiskit/backends/ibm.py [hunk 1]
```diff
@@ -322,6 +322,9 @@ def required_predicates(self) -> List[Predicate]:
     def default_compilation_pass(self, optimisation_level: int = 1) -> BasePass:
         assert optimisation_level in range(3)
         passlist = [DecomposeBoxes()]
+        # If you make changes to the default_compilation_pass,
+        # then please update this page accordingly
+        # https://cqcl.github.io/pytket-qiskit/api/index.html#default-compilation
```
Comment (cqc-melf) [inline pytket/extensions/qiskit/backends/ibm.py:327]: Can you add here the path to the docs file as well?

Comment (CalMacCQ) [reply pytket/extensions/qiskit/backends/ibm.py:327]: Do you mean like this? `pytket-qiskit/docs/intro.txt` ?

Comment (cqc-melf) [reply pytket/extensions/qiskit/backends/ibm.py:327]: yes

Comment (CalMacCQ) [reply pytket/extensions/qiskit/backends/ibm.py:327]: done, thanks

**Review by cqc-melf — APPROVED** [review]

**Pull request merged**